src/pages/templates/page-sections/pricing/threeTierPricing.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  List,
+  ListIcon,
+  ListItem,
+  Text,
+  VStack,
+  Button,
+  Badge,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { FaCheckCircle, FaTimesCircle } from 'react-icons/fa';
+
+export default function PricingTable() {
+  const borderBg = useColorModeValue('gray.200', 'gray.500');
+  const pricingItemStyles = {
+    py: 16,
+    px: 8,
+    mb: 4,
+    mt: { base: 12, lg: 24 },
+    mx: { base: 'auto', lg: 4 },
+    borderColor: borderBg,
+    borderRadius: 10,
+    borderWidth: '2px',
+    maxW: '350px',
+  };
+  const priceProps = {
+    fontSize: '5xl',
+    fontWeight: 'bold',
+    my: 3,
+  };
+  const priceTypeProps = {
+    fontSize: '2xl',
+    fontWeight: 'semibold',
+    color: 'gray.500',
+  };
+  return (
+    <Box m="0 auto" bg={useColorModeValue('gray.50', 'gray.800')}>
+      <Box textAlign="center" mt="1rem">
+        <Heading
+          fontSize="3rem"
+          fontWeight="700"
+          letterSpacing="-0.1rem"
+          mb={8}>
+          Plans that fit your needs.
+        </Heading>
+        <Text fontSize="lg">
+          Start with 14-day free trial. No credit card needed. Cancel at
+          anytime.
+        </Text>
+      </Box>
+
+      <Flex
+        maxW={['100%', '90%']}
+        m="0 auto"
+        direction={['column', 'column', 'row']}
+        textAlign="center">
```

Comment:
Instead of a custom Flex element you can use the `Stack` component here. You can define the direction of the Stack with responsive props. 

Please use the object syntax for responsive values. For example: `direction={{base: 'column', lg: 'row'}}`